cg/services/order_validation_service/workflows/tomte/validation/field_validator.py

```diff
@@ -1,13 +1,17 @@
 from typing import Tuple
-from cg.services.order_validation_service.models.errors import OrderError
+
+from pydantic import ValidationError
+
+from cg.services.order_validation_service.models.errors import ValidationErrors
+from cg.services.order_validation_service.utils import convert_errors
 from cg.services.order_validation_service.workflows.tomte.models.order import TomteOrder
 
 
 class TomteFieldValidator:
 
-    def validate(self, order_json: str) -> Tuple[TomteOrder, list[OrderError]]:
+    def validate(self, order_json: str) -> Tuple[TomteOrder, ValidationErrors]:
```

Comment:
Maybe it should be `TomteOrder | ValidationErrors` or `Tuple[TomteOrder | None, ValidationErrors]`

Reply:
Would it not make more sense to raise the errors then? Is that not the point of a validator?

Reply:
Otherwise, the first option seems reasonable. I will go with that for now unless you object

Reply:
We could raise it if we let the base error class inherit from the exception class but I think it is better to just return it since we might want to apply additional validation. Having validation errors is not really an exceptional thing in this context 😄 Lets go for the first for now